src/main/java/blackjack/ui/BlackjackOutput.java

```diff
@@ -0,0 +1,84 @@
+package blackjack.ui;
+
+import blackjack.domain.Dealer;
+import blackjack.domain.GameResult;
+import blackjack.domain.Participant;
+import blackjack.domain.Player;
+import blackjack.domain.Players;
+import java.util.stream.Collectors;
+
+public class BlackjackOutput {
+
+    private static final String COMMA = ", ";
+    private static final String DEALER_NAME = "딜러와 ";
+    private static final String ENTER = "\n";
+    private static final String DEALER_DRAW = "딜러는 16이하라 한장의 카드를 더 받았습니다.";
+
+    public static void printDrawCard(Players players) {
+        String playerName = players.getPlayers().stream()
+            .map(Player::getName)
+            .collect(Collectors.joining(COMMA));
+
+        StringBuilder stringBuilder = new StringBuilder();
+        stringBuilder
+            .append(DEALER_NAME)
+            .append(playerName)
+            .append("에게 2장을 나누었습니다.");
+        System.out.println(stringBuilder);
+    }
+
+    public static void printAllCard(Players players, Dealer dealer){
+        players.getPlayers().stream()
```

Comment:
```suggestion
        players.getPlayers()
```
`stream`도 없어도 괜찮을 것 같습니다 : )